app/presenters/blacklight/document_presenter.rb

```diff
@@ -69,7 +69,7 @@ def display_type(base_name = nil, default: nil)
     # @param [Hash] options
     # @option options [String] :value
     def field_value field_config, options = {}
-      field_presenter(field_config, options).render
+      field_config ? field_presenter(field_config, options).render : ""
```

Comment:
what if this nil check was moved up into `heading` (line 44)?  I'm not sure we should allow calling field_value with a nil field_config.

Reply:
Yeah that fine.  I think I moved it here merely because I could make a one liner :)

Reply:
Sure thing 👍